cli/commands/ListCustomersCommand.php

```diff
@@ -43,15 +42,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         //Array of customer-table
         $customers = CustomerQuery::create()->find();
+        $output->writeln("Currently there is a total of ".count($customers)." customers\n");
 
-        echo "Currently there is a total of ".count($customers)." customers\n";
-
-        //Runs through the customer-table to initialise and list up all customer_firstName values
+        $table = new Table($output);
+        $table->setHeaders(["Firstname", "Lastname", "ZIP", "City", "Country"]);
         foreach ($customers as $customer)
         {
-            echo $customer->getFirstName()."\n";
+                //The values of the customer-table will be viewed in table format
+                $table->setRows([
+                    [$customer->getFirstName(), $customer->getLastName(), $customer->getZip(), $customer->getCity(), $customer->getCountry()],
+                ]);
+            $table->render();
```

Comment:
I think this needs to changed because you overwrite the rows multiple times if there are more customers, and the table is also rendered multiple times.